tests/core/gui/test_new_version.py

```diff
@@ -399,4 +408,64 @@ def test_windows_update_newest_announced_is_draft():
 )
 @patch("parsec.core.gui.new_version.QSysInfo.currentCpuArchitecture", new=mocked_CPU(64))
 def test_windows_update_newest_announced_is_malformed():
-    assert smallcheck() == ((1, 9, 0), "https://github.com/Scille/parsec-cloud/releases/latest")
+    assert smallcheck() == ((1, 9, 0), gui_check_version_url)
+
+
+@patch("parsec.core.gui.new_version.__version__", new="1.7.0")
+@patch(
+    "parsec.core.gui.new_version.urlopen",
+    new=urlopener(
+        head_version="1.9.0",
+        api_json={
+            "v1.9.0-rc3": {"draft": False, "prerelease": False, "32": True, "64": True},
```

Comment:
is it expected to have `prerelease: False` for a rc release ? (and also `32: True, 64: True` ?)

Reply:
No, it is expected to have `prerelease: True` for a prerelease, from RC to alpha. But as doing a manual release as obligated by hardware signing is a boring, simple process, it is also expected that we might end up forgetting to set that tiny checkbox at the end of it one day or another. So this PR is made to work whether we forget that checkbox or not, and we also test that :)

Reply:
Ha ! I didn't read the test's name 😄